topo_processor/util/file_converter.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from topo_processor.cog.command import Command
```

Comment:
The `command` module is not `COG` specific anymore. I would move it to somewhere more generic (a `system` utils for example).